freebsd/Platform.h

```diff
@@ -26,9 +26,13 @@ extern const SignalItem Platform_signals[];
 
 extern const unsigned int Platform_numberOfSignals;
 
+extern const MeterClass* const Platform_meterTypes[];
+
 void Platform_setBindings(Htop_Action* keys);
```

Comment:
Shouldn't this go after the `Platform_done` function for consistent order?

Reply:
Heh.  This was a of case of a variable being intermingled with function declarations, while all other variables were declared together near the start of the header, so I shifted this variable declaration together with the others for consistent ordering :P

Reply:
I was referring to the `Platform_setBindings` declaration, because on all other platforms this one comes after the `Platform_done` declaration.

Reply:
@BenBE ah I see - yep, will tweak that to be more consistent.